scripts/local-dev-setup.sh

```
@@ -0,0 +1,342 @@
+    npx ts-node scripts/simple-seed.ts
+    
+    print_success "Database setup completed!"
+}
+
+# Start application
+start_application() {
+    print_status "Starting ChemChat application..."
+    
+    # Set environment
+    export NODE_ENV=development
+    export $(cat .env.local | grep -v '^#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)